TodoSPA/App/Scripts/todoListCtrl.js

```diff
@@ -31,6 +31,11 @@ angular.module('todoApp')
             $scope.todoList = results;
             $scope.loadingMessage = "";
         }).error(function (err) {
+            if (err.access_token)
+            {
+                // Re-request the token with additional claims
+                adalService.acquireTokenRedirect('/api/TodoList', null, JSON.stringify(err));
```

Comment:
There may be an extraction step necessary here unless the service is sending back only the `claims` parameter here. If this works, ignore this comment.